sqldelight-gradle-plugin/src/test/fixtures/keyword_identifiers/src/main/sqldelight/com/sample/Test.sq

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE test (
+  "DESC" TEXT,
+  `DESC` TEXT,
+  [DESC] TEXT AS java.util.List
```

Comment:
Maybe eventually we special case Java collections? It's not important, was just a passing thought.